librz/analysis/xrefs.c

```diff
@@ -21,7 +21,8 @@ refs 10->20 C 16->10 J 20->10 C
 // XXX: is it possible to have multiple type for the same (from, to) pair?
 //      if it is, things need to be adjusted
 
-static RzAnalysisXRef *rz_analysis_xref_new(ut64 from, ut64 to, ut64 type) {
+static RzAnalysisXRef *
```

Comment:
Please remove this unrelated change from the PR

Reply:
sure. Done.